### PR TITLE
Fixes #360 -- Install tanner using in a virtualenv

### LINKED ISSUE #360: Make a virtual environment for tanner
Please move to a virtual environment for running tanner. This will require changing the installation directory of tanner from /opt/tanner. @glaslos @afeena can suggest what would be the right way to do the same.

### PR DESCRIPTION
**Pull request by Parth1811**
Step to install:-

1. Make a virtual environment
2. `pip install -r requrement.txt`
3. Go to the Tanner DIrectory
4. `pip install -e .`
5. Run `tanner`

Note:- this is not a full implementation there are some changes and tests to be worked on, just wanted to get a review on this.

### CODE REVIEW

File: tanner/config.py [hunk 1]
```diff
@@ -3,22 +3,23 @@
 import os
 import sys
 
+TANNER_DIR = os.path.expanduser("~/.tanner")
```
Comment (afeena) [inline tanner/config.py]: And what if I want to setup TANNER_DIR? I should always change it in the code?

Comment (Parth1811) [reply tanner/config.py]: we can have an option while setup to change the default directory.........should we do this? is there a need for this?

Comment (afeena) [reply tanner/config.py]: if it's possible and easy to do, I definitely prefer to have a control on folders usage :)

**Comment by afeena** [comment]
@Parth1811 Could you please update me with the status of this PR? :)

**Comment by afeena** [comment]
Closed since no updates

**Pull request closed without merging**